From 912510392373321e81ecad6218ac0efc58069b81 Mon Sep 17 00:00:00 2001
From: Philip Mueller <philm@manjaro.org>
Date: Fri, 31 Jul 2026 21:53:24 +0200
Subject: [PATCH 044/176] [FROM-ML] iio: imu: bmi270: add BOSC0160 PNP ID for
 gaming handhelds

Adding BOSC0160 as PNP ID will enable more gaming
handhelds using a BMI260 sensor with this PNP ID
to properly use the bmi270 driver.

Cc: stable@vger.kernel.org # 6.18+
Signed-off-by: Philip Mueller <philm@manjaro.org>
---
 drivers/iio/imu/bmi270/bmi270_i2c.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/iio/imu/bmi270/bmi270_i2c.c b/drivers/iio/imu/bmi270/bmi270_i2c.c
index d7faebcbd2fa..839e71b65f05 100644
--- a/drivers/iio/imu/bmi270/bmi270_i2c.c
+++ b/drivers/iio/imu/bmi270/bmi270_i2c.c
@@ -41,6 +41,7 @@ MODULE_DEVICE_TABLE(i2c, bmi270_i2c_id);
 static const struct acpi_device_id bmi270_acpi_match[] = {
 	/* GPD Win Mini, Aya Neo AIR Pro, OXP Mini Pro, etc. */
 	{ "BMI0160",  (kernel_ulong_t)&bmi260_chip_info },
+	{ "BOSC0160", (kernel_ulong_t)&bmi260_chip_info },
 	/* GPD Win Max 2 2023(sincice BIOS v0.40), etc. */
 	{ "BMI0260",  (kernel_ulong_t)&bmi260_chip_info },
 	/* OrangePi Neo 2026 (since BIOS BCPH01_V1.20), etc. */
-- 
2.55.0

